src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMan = people.filter(person =>
+    person.sex === 'm'
+    && (!century || century === Math.ceil(person.died / 100)));
+
+  const avgAge = filteredMan.reduce((total, man) => {
+    const manAge = man.died - man.born;
+
+    return total + manAge;
+  }, 0) / filteredMan.length;
```

Comment:
the process of calculating average age for different people arrays is the same
let's create separate function that will accept people array and will return the average age value